files/es/web/css/_doublecolon_selection/index.md

```diff
@@ -1,56 +1,97 @@
 ---
 title: "::selection"
 slug: Web/CSS/::selection
+l10n:
+  sourceCommit: c6894b701b2297a7523d07855b47605c06f29c91
```

Comment:
This is the commit hash from the `en-US` file in the version we are taking to translate

```suggestion
  sourceCommit: 5fea7c9593f5e4b4ef13ec65064acf1eabf01e4e
```

Reply:
I don't know what happened lol